src/client/src/StreamState.c

```diff
@@ -44,7 +45,7 @@ STATUS fromNewStreamState(UINT64 customData, PUINT64 pState)
     ENTERS();
     STATUS retStatus = STATUS_SUCCESS;
     PKinesisVideoStream pKinesisVideoStream = STREAM_FROM_CUSTOM_DATA(customData);
-    UINT64 state;
+    UINT64 state = STREAM_STATE_NEW;
```

Comment:
To prevent undefined behavior. You're right it doesn't functionally change the code but it does follow the same pattern as other 'from' function points and removed the potential for undefined behavior in future revisions.